pkg/utils/kubeutils/metadata.go

```diff
@@ -8,16 +8,22 @@ import (
 	kubetypes "k8s.io/apimachinery/pkg/types"
 )
 
-func FromKubeMeta(meta metav1.ObjectMeta) *core.Metadata {
```

Comment:
deleting this is breaking, which is fine, but needs to be reflected in the changelog